### ISSUE TITLE
avi videos with xvid compression treated like they're not a thing

### ISSUE THREAD

**Issue by eriksneff** [Radarr/Radarr#1070]
**Description:**

I noticed when using the "missing" filter on the Movies list, that all the movies I see aren't actually missing. I confirmed with about 10 of them that they're all xvid-compressed videos in .avi containers. Thought this might be a bug that these are treated like they're not there when they actually are and do work in plex where I consume them regularly.

**Radarr Version:**
Version 0.2.0.453
Mono Version 4.6.1 (mono-4.6.0-branch-c8sr0/abb06f1 Fri Sep 23 19:24:23 EDT 2016)

**Comment by fryfrog** [comment]
For what it is worth, AVI files have been parsing for me w/o issue.

![avi working](https://cloud.githubusercontent.com/assets/87748/23691474/7e566e8a-037d-11e7-8036-b1d00ae2ef7a.PNG)

Are you on Linux? Do you have `libmediainfo` installed?

**Comment by galli-leo** [comment]
If this is still an issue, please let me know and I will reopen this.

**Comment by eriksneff** [comment]
Hi Leo! ( @galli-leo ) - quick off-topic - thanks for the wonderful work on Radarr - it's WAYYYY better than couchpotato, the new hot shit in my opinion!

Ok, back on topic - Sadly, this issue is definitely still happening. I see it was closed as cannot reproduce - fair enough, here's how you can:

A) Some background by way of annotated screenshot that spans plex, radarr, vlc, and finder: 

- https://www.screencast.com/t/WAGV9xnj


B) Links to files of the xvid-encoded-avi-containered variety that Radarr refuses to index:

1. http://www.filehosting.org/file/details/658184/sph-alfranken.avi
2. http://www.filehosting.org/file/details/658192/Zoolander%20KLAXXON.avi


My guess is you'll be able to reproduce once you try those files. And if you can't, then well, maybe it's a mono/mac-specific bug situation? Not sure, but I'd be surprised if this will be something we'll be having to explain, as I think it's pretty likely you'll be able to reproduce.

**Comment by galli-leo** [comment]
@eriksneff Can you please try and execute the Housekeeping Task under System -> Tasks?

**Comment by galli-leo** [comment]
Also if you do a disk scan what do the logs say?

**Comment by eriksneff** [comment]
@galli-leo I manually ran the Housekeeping Task (it was on a schedule and had been last run 8hrs ago, so probably no improvement there I'd guess), then I navigated to zoolander, hit the refresh info and rescan disk, and then got this from the logs:

17-4-19 13:19:26.3|Info|RefreshMovieService|Updating Info for Zoolander
17-4-19 13:19:27.0|Info|DiskScanService|Scanning disk for Zoolander
17-4-19 13:19:27.1|Warn|ParsingService|Unable to parse movie info from path /Volumes/eeeeeeatme/complete/movies/movies/zoolander/Zoolander KLAXXON.avi
17-4-19 13:19:27.1|Info|DiskScanService|Completed scanning disk for Zoolander

Any luck reproducing on your end?

**Comment by galli-leo** [comment]
Rename the file to Zoolander 2001.avi

**Comment by eriksneff** [comment]
Yup yup, see update to the post above...

**Comment by galli-leo** [comment]
@eriksneff Sorry not seeing the update regarding renaming to "Zoolander 2001.avi"? Did that work?

**Comment by eriksneff** [comment]
No worries, I'll take it out of the previous post and put it here (hint: yes, that worked):

**UPDATE:** I have a solution - the issue appears to have been the matchability of the file names, not the encoding of file itself. I renamed this:

movies\zoolander\zoolander-klaxxon.avi
to
movies\zoolander\zoolander.avi
and got nothing

but then I renamed the folder and file to include the year:
movies\zoolander (2001)\zoolander (2001).avi
and presto, the file was detected.

In retrospect, I think the reason it didn't occur to me that this could be the issue was that Plex was detecting the file, so I didn't think to double-check the folder and file naming.

Regardless, perhaps this issue implies a user need for a new ui feature that communicates file scanning/matching failures, and/or some improvements to name matching, but I digress...
